website/docusaurus.config.js

```diff
@@ -148,6 +148,17 @@ const lastVersion = versions[0];
         ],
       },
     ],
+    [
+      'content-docs',
+      /** @type {import('@docusaurus/plugin-content-docs').Options} */
+      ({
+        id: 'contributing',
```

Comment:
Some options of the main versioned docs instance are missing, is this on purpose?

```js
        showLastUpdateAuthor: false,
        showLastUpdateTime: true,
        editUrl:
          'https://github.com/facebook/react-native-website/blob/master/website/',
        remarkPlugins: [require('@react-native-website/remark-snackplayer')],
```

As I'm also adding another docs instance in https://github.com/facebook/react-native-website/pull/3000, maybe it should later be refactored so that those docs options are shared over instances?